server/sonar-web/src/main/js/apps/projects/store/utils.js

```diff
@@ -34,16 +34,18 @@ const getAsLevel = value => {
   return null;
 };
 
+// TODO Maybe use parseAsString form helpers/query
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)